lib/playground.dart

```diff
@@ -202,20 +203,24 @@ class Playground implements GistContainer, GistController {
     return new Future.value();
   }
 
-  Future shareAnon() {
-    return gistLoader.createAnon(mutableGist.createGist()).then((Gist newGist) {
-      editableGist.setBackingGist(newGist);
-      overrideNextRoute(newGist);
-      router.go('gist', {'gist': newGist.id});
-      var toast = new DToast('Created ${newGist.id}')..show()..hide();
-      toast.element
-        ..style.cursor = "pointer"
-        ..onClick.listen((e)
-            => window.open("https://gist.github.com/anonymous/${newGist.id}", '_blank'));
-    }).catchError((e) {
-      String message = 'Error saving gist: ${e}';
-      DToast.showMessage(message);
-      ga.sendException('GistLoader.createAnon: failed to create gist');
+  Future shareAnon() async{
+    Future<String> f = _formSummary();
```

Comment:
So, we don't need to mix `async` style and  `Futures` style. Since this code is using Futures, lets keep it that way for now.we can:

- remove the `async` keyword on the method
- name `f` something better (which was my fault :) )
- change the `await` keyword below to a `return`
- fix the indent in the method (this link might help https://www.dartlang.org/articles/style-guide/#whitespace). It might be easiest to format it first w/o worrying about wrapping long lines, then wrap the one or two that are long